runtime/sema/authaccount_type.go

```diff
@@ -277,6 +284,32 @@ var AuthAccountTypeLoadFunctionType = func() *FunctionType {
 	}
 }()
 
+const authAccountTypeTypeAtFunctionDocString = `
+Reads the type of an object from the account's storage which is stored under the given path, or nil if no object is stored under the given path.
+
+If there is an object stored, the type of the object is returned without modifying the stored object. 
+
+The path must be a storage path, i.e., only the domain ` + "`storage`" + ` is allowed
+`
+
+var AuthAccountTypeTypeAtFunctionType = func() *FunctionType {
+
+	return &FunctionType{
+		Parameters: []*Parameter{
+			{
+				Label:          ArgumentLabelNotRequired,
+				Identifier:     "path",
+				TypeAnnotation: NewTypeAnnotation(StoragePathType),
+			},
+		},
+		ReturnTypeAnnotation: NewTypeAnnotation(
+			&OptionalType{
+				Type: MetaType,
+			},
+		),
+	}
+}()
```

Comment:
No need for an immediately invoked closure here:

```suggestion
var AuthAccountTypeTypeAtFunctionType = &FunctionType{
	Parameters: []*Parameter{
		{
			Label:          ArgumentLabelNotRequired,
			Identifier:     "path",
			TypeAnnotation: NewTypeAnnotation(StoragePathType),
		},
	},
	ReturnTypeAnnotation: NewTypeAnnotation(
		&OptionalType{
			Type: MetaType,
		},
	),
}
```